build-system/server/new-server/transforms/utilities/script.ts

```diff
@@ -27,18 +30,27 @@ export interface ScriptNode extends PostHTML.Node {
   };
 }
 
+const VALID_SCRIPT_EXTENSION = ['.js', '.mjs'];
```

Comment:
Readability nit: Since this is an array, make it plural (`VALID_SCRIPT_EXTENSIONS`).

Reply:
done.